### DISCUSSION TITLE
次字幕secondary-sid双语

### DISCUSSION THREAD

**Discussion by sqq135** [hooke007/mpv_PlayKit#421] [Q&A 问答] [answered]
想打开双语模式，也就是上下各一个字幕，但是发现两个字幕重合了，安装个lazy在其他位置，还是不行，字幕重合，不知道怎么解决，感觉是个很简单的问题，但就是没找到，希望有大佬提点一句，或者告诉我哪里找就行，mpv.io 的secondary内容也没提到啊

**Comment by hooke007** [comment, marked as answer]
建议上图或者日志，我默认当你说的是图形字幕，图形字幕不支持改变位置

**Reply by sqq135** [reply]
没错，是我傻了，就是图形字幕，我以前也花了功夫弄明白了，后来又给忘了。你一说想起来了。

另外，次字幕的位置参数是这个--secondary-sub-pos =

话题结束。
